client/client-lib/src/lib.rs

```diff
@@ -243,6 +243,21 @@ impl<T: AsRef<ClientConfig> + Clone> Client<T> {
         Ok(OutPoint { txid, out_idx: 0 })
     }
 
+    /// Validate tokens without claiming them. This function checks if signatures are valid
```

Comment:
Throughout the comments and function naming, I used the term `tokens` instead of `coins` since I think it was mentioned on the devcall that the correct term is `tokens`. It may be confusing since other functions refer to `coins`. Let me know if I should rename anything.